vault/resource_gcp_secret_impersonated_account.go

```diff
@@ -200,6 +206,10 @@ func gcpSecretImpersonatedAccountUpdateFields(d *schema.ResourceData, data map[s
 	if v, ok := d.GetOk(consts.FieldTokenScopes); ok {
 		data[consts.FieldTokenScopes] = v.(*schema.Set).List()
 	}
+
+	if v, ok := d.GetOk(consts.FieldTTL); ok {
```

Comment:
We should also handle this on Read (gcpSecretImpersonatedAccountRead) so that we can set it to state.

Reply:
Just pushed a [commit](https://github.com/hashicorp/terraform-provider-vault/pull/2318/commits/6d492bf7407e0dde9727363e7b9b2136a760826a), I hope this is the correct way